views/components/BetaSignUpForm.js

```diff
@@ -1,63 +1,62 @@
 import React from 'react';
 import FormField from './FormField';
 
-const BetaSignUpForm = props => {
+const BetaSignUpForm = ({ optin, betaCount }) => {
+  let fieldViews = [];
+  let defaultBetaCount = 3;
+  // If a specific number of referral/beta fields are given. Check if betaCount is
+  // 1 and singularize the header. Also loop over beta count and add FormFields.
+  let PluralOrSingularFriends = betaCount == 1 ? 'Friend' : 'Friends';
```

Comment:
Can we lowercase to `pluralOrSingularFriends`? Traditionally I've understood capitalized variable names to indicate classes or services. For regular variables, we should lowercase em.